src/components/network/line-layer.js

```diff
@@ -81,24 +81,32 @@ class LineLayer extends CompositeLayer {
 
     renderLayers() {
         const layers = [];
-
         // lines : create one layer per nominal voltage, starting from higher to lower nominal voltage
         this.state.compositeData.forEach(compositeData => {
             const color = this.props.getNominalVoltageColor(compositeData.nominalVoltage);
 
+            const doDash = line => {
```

Comment:
can you make it symmetric between the "line.p1 == null || line.p2 == null" and the "line.p1==null && line.p2 == null" ?
Either make 2 functions doDash and isDisconnected or inline both in their respective callsite.
Just for symmetry and aesthetics